packages/ui-react/src/components/FinalizePayment/FinalizeStripePpvPayment.tsx

```diff
@@ -0,0 +1,37 @@
+import { useCallback, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getModule } from '@jwp/ott-common/src/modules/container';
+import CheckoutController from '@jwp/ott-common/src/controllers/CheckoutController';
+
+const FinalizeStripePpvPayment = () => {
+  const checkoutController = getModule(CheckoutController);
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const finalize = useCallback(async (paymentIntent: string) => {
+    try {
+      await checkoutController.finalizeStripePpvPayment(paymentIntent);
```

Comment:
Did you try to reuse everything in FanalizeAdyenPayment except using a different method instead of a `checkoutController.finalizeAdyenPayment` one?

Reply:
I did think it through and couldn't think of anything that would improve the reusability further than what it is with this PR. Do you have anything specific on your mind?